pkg/cloud/instance_test.go

```diff
@@ -187,37 +189,6 @@ var _ = Describe("Instance", func() {
 				Should(MatchError(unknownErrorMessage))
 		})
 
-		// The folloing test checks that DomainID and Account are set (or not) in the DeployVirtualMachineParams
-		// interface passed to DeployVirtualMachine.
-		describeDomainAccountTest := func(desc string) func(string, string) string {
-			return func(account string, domainID string) string {
-				return fmt.Sprintf(`"%s" and "%s", %s`, account, domainID, desc)
-			}
-		}
-		DescribeTable("DomainID and Account test table.",
```

Comment:
We no longer need to set domain and account, so all these extra tests are for code that doesn't exist.